scripts/install_macos.sh

```diff
@@ -1,27 +1,21 @@
 #!/bin/bash
 # Install Homebrew
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+set -e
 
-# Install dependencies
-brew install gmp coreutils python3 pipx
-
-# Install Elan
-# At startup, Bash sources .bash_profile, or if that doesn't exist, .profile.
-# Elan adds itself to the PATH in .bash_profile, or if that doesn't exist, .profile.
-# pipx only adds itself to the PATH in .bash_profile.
-# So we will create .bash_profile if it doesn't exist yet, ensuring .profile still gets loaded after installing pipx.
-if ! [ -r ~/.bash_profile ]; then
-echo '[ -r ~/.profile ] && source ~/.profile' >> ~/.bash_profile
+if ! which brew; then
```

Comment:
```suggestion
if ! which brew > /dev/null; then
```
Without this, it'll print output to stdout, which we probably don't need to see.